code/modules/multiz/movement.dm

```diff
@@ -89,6 +90,16 @@
 		return
 	. = ..()
 	if(.)
+		// Chance to fail while trying to climb down a lattice structure or ladder
+		var/turf/T = GetAbove(src)
+		var/area/A = get_area(src)
+		if (locate(/obj/structure/lattice, T) || locate(/obj/structure/ladder, T))
```

Comment:
If this is only meant to happen when you move downwards, you'll need a check for `direction == DOWN` aswell.

Reply:
Added check.